packages/tools/src/stateManagement/annotation/annotationState.ts

```diff
@@ -251,7 +239,32 @@ function getAnnotation(annotationUID: string): Annotation {
  */
 function removeAllAnnotations(): void {
   const manager = getAnnotationManager();
-  manager.removeAllAnnotations();
+  const removedAnnotations = manager.removeAllAnnotations();
+
+  for (const annotation of removedAnnotations) {
+    triggerAnnotationRemoved({
+      annotation,
+      annotationManagerUID: manager.uid,
+    });
+  }
+}
+
+/**
+ * Removes all annotations associated with the specified group (FrameOfReferenceUID) and tool, or
+ * all annotations for the group (FrameOfReferenceUID) if the tool name is not provided.
+ * @param groupKey - The group key to remove annotations for (in default manager it is FrameOfReferenceUID).
+ * @param toolName - Optional. The name of the tool to remove annotations for.
+ */
+function removeAnnotations(groupKey: string, toolName?: string): void {
+  const manager = getAnnotationManager();
+  const removedAnnotations = manager.removeAnnotations(groupKey, toolName);
+
+  for (const annotation of removedAnnotations) {
```

Comment:
I believe it is meant to be AnnotationGroupSelector instead of string? I will make this change as that is how other functions in this file handle this.